website/versioned_docs/version-5.x/ecosystem-css.md

```diff
@@ -0,0 +1,336 @@
+
+## Shared CSS
+
+It is best for both performance and developer experience to have some shared CSS. Often, the shared CSS is part of a "styleguide" or "design system."
+
+Sometimes the design system is created in-house by a company, and other times it's an open source design system that is available on npm (Material UI, Bootstrap, Semantic UI, etc). For both cases, it's important that there is only a single copy of the CSS on the page at any time. When using [the recommended setup](/docs/recommended-setup), this is accomplished by following [the techniques in this documentation](http://localhost:3000/docs/recommended-setup#sharing-with-import-maps).
+
+### In-House Design System
+
+Our recommendation for in-house design systems is to create a [utility microfrontend](/docs/module-types#utilities) (often named `@your-org-name/styleguide`). Contained within the utility microfrontend are shared CSS and javascript components that are available for all other microfrontends to use.
+
+Other microfrontends can access shared javascript components via [cross-microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports), and apply shared, global css classes to their components in the normal way (`<div class="bold">`).
+
+Here are some examples:
+
+- https://github.com/react-microfrontends/styleguide
+- https://github.com/vue-microfrontends/styleguide
+- https://github.com/polyglot-microfrontends/styleguide
+
+The alternative to creating a utility microfrontend for your styleguide is to publish it to npm. The drawback to this approach is that it makes it easier to have duplicate copies of the styleguide, and also easier to have multiple versions of the styleguide. Npm packages are not independently deployable, nor are they singletons, but for a styleguide it's often desireable that it's centrally managed and can be deployed separately from the microfrontends that use them.
+
+### Third Party Design System
+
+When using a third-party design system, such as Material UI, Bootstrap, Semantic, etc, it is important that only one copy and version of the design system is loaded on the page. To accomplish this, here are two implementation options.
+
+1. Add the design system libraries to your SystemJS import map, then mark them as external ([full documentation](/docs/recommended-setup#sharing-with-import-maps)). Alternatively, do the equivalent with [module federation](/docs/recommended-setup#sharing-with-module-federation).
+1. Create a utility microfrontend (often called `@your-org-name/styleguide`) that contains all shared css and javascript components. Re-export the components from the design system so that all other microfrontends can access them via [cross microfrontend import](/docs/recommended-setup#cross-microfrontend-imports) (`import { Button } from '@your-org-name/styleguide';`).
```

Comment:
```suggestion
1. Create a utility microfrontend (often called `@your-org-name/styleguide`) that contains all shared CSS and Javascript components. Re-export the components from the design system so that all other microfrontends can access them via [cross microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports) (`import { Button } from '@your-org-name/styleguide';`).
```